python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -2260,6 +2260,7 @@ def get_hashable(self) -> Hashable:
             self.name,
             json.dumps(self.options),
             tuple(self.schema.items()),
+            self._ctor_arguments(self.children)[1:],
```

Comment:
I initially tried replacing this with `(*super().get_hashable(), self.name, json.dumps(self.items))` to reuse the base class's implementation of hashing schema + children and only adding the extra bits. But mypy didn't like the types, even with all the return types declared as `tuple[Hashable, ...]`